### PR TITLE
configmap.go: add one initialization symbol

### PR DESCRIPTION
**Pull request by zhangxiaoyu-zidif**
add one initialization symbol

### CODE REVIEW

File: pkg/kubectl/configmap.go [hunk 1]
```diff
@@ -196,7 +196,7 @@ func handleConfigMapFromFileSources(configMap *api.ConfigMap, fileSources []stri
 				}
 			}
 		} else {
-			err = addKeyFromFileToConfigMap(configMap, keyName, filePath)
+			err := addKeyFromFileToConfigMap(configMap, keyName, filePath)
```
Comment (deads2k) [inline pkg/kubectl/configmap.go]: While you're here, scope to the `if err :=...; err != nil`

Comment (zhangxiaoyu-zidif) [inline pkg/kubectl/configmap.go]: @deads2k Thanks for your guide, I repair it.=)

**Comment by zhangxiaoyu-zidif** [comment]
@deads2k  Thanks, squash for merge already.

**Comment by deads2k** [comment]
@k8s-bot ok to test

**Pull request merged**